spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/configuration/JobDependencies.java

```diff
@@ -83,16 +83,7 @@
 import org.springframework.cloud.dataflow.server.service.TaskJobService;
 import org.springframework.cloud.dataflow.server.service.TaskSaveService;
 import org.springframework.cloud.dataflow.server.service.TaskValidationService;
-import org.springframework.cloud.dataflow.server.service.impl.ComposedTaskRunnerConfigurationProperties;
-import org.springframework.cloud.dataflow.server.service.impl.DefaultLauncherService;
-import org.springframework.cloud.dataflow.server.service.impl.DefaultTaskDeleteService;
-import org.springframework.cloud.dataflow.server.service.impl.DefaultTaskExecutionInfoService;
-import org.springframework.cloud.dataflow.server.service.impl.DefaultTaskExecutionRepositoryService;
-import org.springframework.cloud.dataflow.server.service.impl.DefaultTaskExecutionService;
-import org.springframework.cloud.dataflow.server.service.impl.DefaultTaskJobService;
-import org.springframework.cloud.dataflow.server.service.impl.DefaultTaskSaveService;
-import org.springframework.cloud.dataflow.server.service.impl.TaskAppDeploymentRequestCreator;
-import org.springframework.cloud.dataflow.server.service.impl.TaskConfigurationProperties;
+import org.springframework.cloud.dataflow.server.service.impl.*;
```

Comment:
Please resolve checkstyle issues for a clean compile.

Reply:
OK